rs/nervous_system/integration_tests/tests/sns_release_qualification.rs

```diff
@@ -267,4 +271,40 @@ fn test_sns_upgrade(sns_canisters_to_upgrade: Vec<SnsCanisterType>) {
             sns_governance_canister_id,
         );
     }
+
+    // Check the SNS canisters' wasm_memory_limits
+    // TODO(NNS1-3278): Move this check to before upgrading the canisters in the
+    // non-release-qualification tests.
+    check_memory_limits_set(pocket_ic, root_canister_id);
```

Comment:
Can we move this outside of this PR into its own, please?

Reply:
Done